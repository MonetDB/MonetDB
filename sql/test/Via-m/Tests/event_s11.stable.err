stderr of test 'event_s11` in directory 'sql/test/Via-m` itself:


# 15:59:49 >  
# 15:59:49 >   mserver5 "--config=/export/scratch0/rijke/monetdb20090918/etc/monetdb5.conf" --debug=10 --set gdk_nr_threads=0 --set "monet_mod_path=/export/scratch0/rijke/monetdb20090918/lib/MonetDB5:/export/scratch0/rijke/monetdb20090918/lib/MonetDB5/lib:/export/scratch0/rijke/monetdb20090918/lib/MonetDB5/bin" --set "gdk_dbfarm=/export/scratch0/rijke/monetdb20090918/var/MonetDB5/dbfarm"  --set mapi_open=true --set xrpc_open=true --set mapi_port=32412 --set xrpc_port=40006 --set monet_prompt= --trace  "--dbname=mTests_src_test_Via-m" --set mal_listing=0 "--dbinit= include sql;" ; echo ; echo Over..
# 15:59:49 >  

MAPI  = (monetdb) /var/tmp/mtest-287082/.s.monetdb.35211
QUERY = select add_event('event 4', 123456, 'artist2', 'location3', 'description3');
ERROR = !types int(17,0) and date(0,0) are not equal
CODE  = 42000
MAPI  = (monetdb) /var/tmp/mtest-30274/.s.monetdb.37685
QUERY = delete from event where event_name = 'event 2';
ERROR = !DELETE: FOREIGN KEY constraint 'event_metadata.fk_event_metadata_event_id' violated
CODE  = 40002
MAPI  = (monetdb) /var/tmp/mtest-225531/.s.monetdb.33874
QUERY = insert into event_metadata (
            event_id,
            metadata_name,
            metadata_value )
        values ( 1,
                'artist',
                'artist1' );
ERROR = !INSERT INTO: UNIQUE constraint 'event_metadata.unq_metadata' violated
CODE  = 40002
MAPI  = (monetdb) /var/tmp/mtest-287082/.s.monetdb.35211
QUERY = select add_event('event 4', '2009-06-24', 'artist4', 'location4', 'description4');
ERROR = !INSERT INTO: UNIQUE constraint 'event.unq_event_name' violated
CODE  = 40002

# 15:59:49 >  
# 15:59:49 >  Done.
# 15:59:49 >  

