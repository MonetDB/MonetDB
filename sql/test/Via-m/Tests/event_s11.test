statement error
select add_event('event 4', 123456, 'artist2', 'location3', 'description3')

query I nosort
select add_event('event 4', '2009-06-23', NULL, NULL, 'description3')
----
5

statement error
delete from event where event_name = 'event 2'

statement error
insert into event_metadata (
    event_id,
    metadata_name,
    metadata_value )
values ( 1,
        'artist',
        'artist1' )

statement error
select add_event('event 4', '2009-06-24', 'artist4', 'location4', 'description4')

