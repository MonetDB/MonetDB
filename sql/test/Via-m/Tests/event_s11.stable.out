stdout of test 'event_s11` in directory 'sql/test/Via-m` itself:


# 10:49:40 >  
# 10:49:40 >  "mserver5" "--debug=10" "--set" "gdk_nr_threads=0" "--set" "mapi_open=true" "--set" "mapi_port=36838" "--set" "mapi_usock=/var/tmp/mtest-5263/.s.monetdb.36838" "--set" "monet_prompt=" "--forcemito" "--dbpath=/ufs/sjoerd/@Monet-stable/var/MonetDB/mTests_sql_test_Via-m"
# 10:49:40 >  

#select add_event('event 4', '2009-06-23', NULL, NULL, 'description3');
% . # table_name
% %1 # name
% int # type
% 1 # length
[ 5	]


# 10:49:40 >  
# 10:49:40 >  "mclient" "-lsql" "-ftest" "-Eutf-8" "-i" "-e" "--host=/var/tmp/mtest-5263" "--port=36838"
# 10:49:40 >  

#declare id_event integer;
#set id_event = -1;
#declare id_event2 integer;
#set id_event2 = add_event('event 4', '2009-06-23', NULL, NULL, 'description3');
#declare id_event3 integer;

# 10:49:40 >  
# 10:49:40 >  "Done."
# 10:49:40 >  

